.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,8 @@
 <!--- Describe the changes here. --->
 
-**Changelog-friendly one-liner**: <!-- One-liner description here -->
-
 ##### Contributor checklist
 
 - [ ] Provided the tests for the changes.
-- [ ] Gave a clear one-line description in the PR (that the maintainers can add to CHANGELOG.md on release).
+- [ ] Assure PR title is short, clear, and good to be included in the changelog
```

Comment:
```suggestion
- [ ] Assure PR title is short, clear, and good to be included in the user-oriented changelog
```

Reply:
@atugushev WTDY? Commit suggestion if you agree and lets move on.